package.json

```diff
@@ -6,8 +6,13 @@
   "scripts": {
     "test": "cypress run",
     "test:open": "npm run lint && cypress open",
+    "test:pre": "npm run reports:clean",
+    "test:post": "npm run combine:reports && npm run generate:reports",
```

Comment:
the pre and post hooks should be declared in this way: pre<scriptName> and post<scriptName>, so right values are:

- pretest
- posttest

Reply:
![image](https://user-images.githubusercontent.com/92124971/182970203-bd01b5a4-1cb3-4e32-a736-68c8a46e78f2.png)
![image](https://user-images.githubusercontent.com/92124971/182970233-4e597015-96e2-4ce4-a6e7-d3421347ea75.png)